fem/tmop.hpp

```diff
@@ -1183,6 +1185,8 @@ class TMOP_Integrator : public NonlinearFormIntegrator
 
    DiscreteAdaptTC *discr_tc;
 
+   MemoryType pa_mt = MemoryType::DEFAULT;
+
```

Comment:
This is already inherited from the base class. Or am I missing something?

Reply:
@artv3 and/or @tomstitt, can we just remove this variable and the method below.

Reply:
Removed, thanks @v-dobrev !